modules/web/package.json

```diff
@@ -52,7 +52,6 @@
     "jquery-ui": "^1.12.1",
     "js-yaml": "^3.7.0",
     "jsplumb": "^2.2.8",
-    "jstree": "^3.3.2",
```

Comment:
@hemikak ; importing jstree and its style.css has broken the stylings of the type mapper, so this has been reverted back to previous state to load from requirejs until this is properly being fixed.